pkg/manifestgen/manifestgen.go

```diff
@@ -49,6 +49,11 @@ type Options struct {
 	// generate an error.
 	WarningsOutput io.Writer
 
+	// DepsolveOutput will receive any warnings that are part of
+	// the depsolving step. If it is unset output ends up on the
+	// default stdout/stderr.
+	DepsolveOutput io.Writer
```

Comment:
Nitpic: for consistency, would it make sense to call this `DepsolveWarnings` or `DepsolveWarningsOutput` (which is quite long :confused: ) instead? The actual depsolve output is where we get the response from the depsolve tool.

Reply:
Yea I kinda debated that, I dunno. If others feel strongly I'm fine with renaming it. The `*Warnings` things also carry (some) meaning in that in other places those would be direct `error`'s if not redirected.

Reply:
Sure, but ultimately, the writer will be used for the solver's `Stderr` output, not its `Stdout`. This is my main concern with naming it `...Output`.

Reply:
So you feel strongly. It's been renamed.